injected/Overlay.js

```diff
@@ -160,7 +160,7 @@ function getHighlightColor(color) {
 
 var Overlay = {
 
-  highlightComponentInstance: function(instance, name, config) {
```

Comment:
This is intentionally abstracted this way so that other environments like React Native can override this whole thing and implement highlighting differently.